db/migrate/20210315184532_update_review_time_schema.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateReviewTimeSchema < ActiveRecord::Migration
+  def change
+    add_column :submission_viewing_events, :total_time, :integer, :default => 0, column_options: { null: false }
```

Comment:
Did we want this to be an integter? Or something else?

Reply:
What else would it be?

Reply:
I was thinking long based on the diagram in Slack. Either works though. I'm fine with it being integer

Reply:
Maybe we ought to use float, I hadn't considered fractional seconds.